### ISSUE TITLE
Vertical nav doesn't convey expanded state

### ISSUE THREAD

**Issue by gnomeontherun** [vmware/clarity#3575]
## Describe the bug

The nav group trigger buttons do not convey their expanded/collapsed states. 

## How to reproduce

https://v1.clarity.design/vertical-nav/icon-links/normal

- Using a screen reader go to https://v1.clarity.design/vertical-nav/icon-links/normal
- Read any of the nav group trigger buttons and observe that the screen reader does not announce the expanded or collapsed state of these controls.

## Expected behavior

Use aria-expanded="true" and aria-expanded="false" to indicate the expanded and collapsed state of these controls

## Versions

**App**

* Angular: [e.g. 6]
* Node: [e.g. 8.10.0]
* Clarity: [e.g. 0.12.5]

**Device:**

* Type: [e.g. MacBook]
* OS: [e.g. iOS]
* Browser [e.g. chrome, safari]
* Version [e.g. 22]

## Additional notes

_Add any other notes about the problem here._

**Comment by gnomeontherun** [comment]
This needs some research to ensure we handle this correctly, I'm not certain where exactly those attributes should go without researching other examples first. @andreiconstantinbytex I'll assign this but look up aria- spec here for expanded and vertical navigation patterns. Sometimes the solution here is that apps need to add it (basically if its an aria attribute that goes on the host or isn't inside of the Clarity component, we might expect the application to handle this instead of internalizing it into Clarity).

**Comment by Jinnie** [comment]
I guess @andreiconstantinbytex needs to comment on this issue directly before we can assign it to him.

**Comment by andreiconstantinbytex** [comment]
Hello, I would like to be assigned to this issue.

**Comment by Jinnie** [comment]
May be ready to close, after we get confirmation that Jeremy's requested text changes for V1 `need merge`/`are merged`/`don't need merge` to V2 and master

**Comment by gnomeontherun** [comment]
This should be closed by #3860

**Comment by Jinnie** [comment]
I think it's in V1 only currently.
@andreiconstantinbytex ?

**Comment by andreiconstantinbytex** [comment]
Indeed, this is in V1 only.
